Python/codegen.c

```diff
@@ -5254,15 +5266,9 @@ codegen_subscript(compiler *c, expr_ty e)
     return SUCCESS;
 }
 
-/* Returns the number of the values emitted,
- * thus are needed to build the slice, or -1 if there is an error. */
 static int
-codegen_slice(compiler *c, expr_ty s)
+codegen_slice_two_parts(compiler *c, expr_ty s)
```

Comment:
I think this can return SUCCESS/ERROR now.

Reply:
Good point.  It doesn't need to return `n` anymore.